StudentList/Fragments/StudentListFragment.cs

```diff
@@ -81,13 +89,18 @@ public override void OnStart()
             base.OnStart();
 
             this.studentAdapter.ItemClick += this.StudentAdapterItemClick;
+            this.subsription = this.store.Select(
```

Comment:
`subscription` name is too general, later if you have more of them you will need to rename it.